### ISSUE TITLE
[deprecation] com.google.firebase.iid FirebaseInstanceIdService

### ISSUE THREAD

**Issue by chenyang88** [flutter/flutter#28982]
```
Launching lib/main.dart on Android SDK built for x86 in debug mode...
Initializing gradle...
Resolving dependencies...
Gradle task 'assembleStagDebug'...
registerResGeneratingTask is deprecated, use registerGeneratedResFolders(FileCollection)
registerResGeneratingTask is deprecated, use registerGeneratedResFolders(FileCollection)
registerResGeneratingTask is deprecated, use registerGeneratedResFolders(FileCollection)
registerResGeneratingTask is deprecated, use registerGeneratedResFolders(FileCollection)
registerResGeneratingTask is deprecated, use registerGeneratedResFolders(FileCollection)
registerResGeneratingTask is deprecated, use registerGeneratedResFolders(FileCollection)
registerResGeneratingTask is deprecated, use registerGeneratedResFolders(FileCollection)
registerResGeneratingTask is deprecated, use registerGeneratedResFolders(FileCollection)
registerResGeneratingTask is deprecated, use registerGeneratedResFolders(FileCollection)
registerResGeneratingTask is deprecated, use registerGeneratedResFolders(FileCollection)

```

**Comment by collinjackson** [comment]
I don't actually see `FirebaseInstanceIdService` in this issue. But anyway, the deprecation warnings in these logs should be resolved with flutter/plugins#1885

**Comment by kroikie** [comment]
@chenyang88

This issue has been moved to https://github.com/FirebaseExtended/flutterfire/issues/400. Any further collaboration will be done there.
